projectName-selenium/pom.xml

```diff
@@ -63,6 +63,49 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
```

Comment:
Please see https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/527/files .
We can adjust only define `dependencyManagement`, it is unnecessary to change selenium project.